helm-charts/chatqna/values.yaml

```diff
@@ -71,6 +71,7 @@ vllm:
   LLM_MODEL_ID: meta-llama/Meta-Llama-3-8B-Instruct
   shmSize: 128Gi
   VLLM_TORCH_PROFILER_DIR: "/tmp/vllm_profile"
+  VLLM_CPU_KVCACHE_SPACE: 40
```

Comment:
This means 40GB sized KV cache: https://docs.vllm.ai/en/latest/getting_started/installation/cpu.html#related-runtime-environment-variables

=> Memory request should be set accordingly (i.e. 40GB + enough memory for the model itself).

Reply:
It would be good to group together all related variables; model ID, KV cache size, and requests, prefixed with e.g. "model specific values"  comment, and the other variables being prefixed with "model independent values" comment.

More complex solution would be including model "profiles" into Helm charts as mappings, and selecting suitable values from that mapping using the specified model ID as a key.

(Llong term solution could be switching to an operator that supports model profiles files or CRDs, and handles these automatically based on selected model and other tags, instead of them being managed with Helm variables.)

Reply:
Seems this setting is specific for DeepSeek model on cpu, I'll move the related settings into cpu-values.yaml, and mark it optional.